llm_assistant/ollama/openai_chat.py

```diff
@@ -0,0 +1,20 @@
+from openai import OpenAI
+import argparse
+# Don't need to add the actual api_key here, just use any string value
```

Comment:
instead of saying what the code does, can we reference why we're doing that in the code? Something like

```python
# We don't need an actual api_key here. See `ollama/README.md`
client = OpenAI(base_url="http://0.0.0.0:8000", api_key="FAKE")
```

Reply:
Thanks a, that makes sense, I have updated the code and the readme.md